src/bin/util/test/mldsa44-import-key-test.sh

```
@@ -0,0 +1,160 @@
+fi
+
+openssl genpkey -algorithm ML-DSA-44 -out "$KEY_FILE"
+
+if ! softhsm2_tool --import "$KEY_FILE" --import-type keypair --id $TOKEN_ID --label test_key --token test0 --pin $TOKEN_PIN >"$IMPORT_OUT" 2>&1; then
+	cat "$IMPORT_OUT"
+  cat "$TOKEN_DIR/token.log"
+	exit 1
+fi
+
+cat "$IMPORT_OUT"
+cat $TOKEN_DIR/token.log
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)